share/spice/seat_geek/events_by_city/seat_geek_events_by_city.js

```diff
@@ -1,17 +1,14 @@
 (function(env) {
     "use strict";
-    env.ddg_spice_seat_geek = function(api_result) {
+    env.ddg_spice_seat_geek_events_by_city = function(api_result) {
 
         if(!api_result || api_result.error || api_result.events.length === 0) {
             return Spice.failed('seat_geek');
         }
 
-        var query = DDG.get_query();
-        var clean_query = query.replace(/((upcoming\s)?(concerts?))|(live(\s(shows?))?)/, '').trim().toLowerCase();
-
-        var months = ['Jan', 'Feb', 'Mar', 'Apr', 'May', 'June', 'July', 'Aug', 'Sept', 'Oct', 'Nov', 'Dec'];
-
-        var days = ['Sunday', 'Monday', 'Tuesday', 'Wednesday', 'Thursday', 'Friday', 'Saturday'];
+        var query = DDG.get_query(),
+            months = ['Jan', 'Feb', 'Mar', 'Apr', 'May', 'June', 'July', 'Aug', 'Sept', 'Oct', 'Nov', 'Dec'],
+            days = ['Sunday', 'Monday', 'Tuesday', 'Wednesday', 'Thursday', 'Friday', 'Saturday'];
 
         Spice.add({
             id: "seat_geek",
```

Comment:
@andrey-p Please change the ids, each one should be unique, so for example this could be "seat_geek_events_by_city", etc.